pubspec.yaml

```diff
@@ -3,14 +3,18 @@ version: 0.9.1+7
 homepage: https://github.com/SunnyApp/flutter_contact
 description: A Flutter plugin to retrieve, create and save contacts and contact-related events on Android and iOS devices.
 maintainer: Eric Martineau
+publish_to: 'none'
```

Comment:
This is added because we want to override the flexidate from https://github.com/agendaboa/flexidate-0.8.0-3 to use the latest timezone dependency.